### DISCUSSION TITLE
Too weak channel separation

### DISCUSSION THREAD

**Discussion by Kabron287** [fhdm-dev/scoppy#40] [General]
Hello, fhdm-dev.
Channels interpenetration is noticeable.
Could you test a solution when before switching from one to another channel, ADC connects to third channel connected to ground.
It may resolve the problem.

**Comment by fhdm-dev** [comment]
> Hello, fhdm-dev. Channels interpenetration is noticeable. Could you test a solution when before switching from one to another channel, ADC connects to third channel connected to ground. It may resolve the problem.

Hi.

I'll see what I can do. It might be as easy as setting the IE (input enable) bit to 0 on the unused ADC GPIOs.
